### ISSUE TITLE
ERROR: cpu exception in dosemu code outside of DPMI client!

### ISSUE THREAD

**Issue by w8l13r** [dosemu2/dosemu2#2179]
**Describe the bug**
A description of what the problem is.

**To Reproduce**
Steps to reproduce the behaviour:

**Attach program/game binaries or provide an URL**
If the problem needs specific DOS files to reproduce, please attach
or provide the download URL.

**Attach the log**
It is located in ~/.dosemu/boot.log
Unless you get the plain crash of dosemu2,
you may need to enable some logging flags
to make your report more useful.
See description of -D option in `man dosemu.bin`.

**A regression?**
If you happened to know this problem didn't
exist on dosemu1 or some earlier versions of
dosemu2, please write.

**Additional info**
Please write here if you did any dosemu2 setup
customizations, like installing any custom DOS
or command.com, or altering the dosemu2
configuration settings. Note that freedos, even
from dosemu-freedos package of dosemu1, counts
as a custom DOS and should be mentioned here.
Also make a note if it is a protected or real-mode
program. If you suspect any particular dosemu2
component (DPMI, PIC, PIT etc) to be responsible
for the problem, please write.

**Comment by w8l13r** [comment]
[boot.log](https://github.com/dosemu2/dosemu2/files/14782451/boot.log)
dosemu is started via terminal and aborts with this message.

"ERROR: cpu exception in dosemu code outside of DPMI client!
Segmentation fault (Invalid permissions for mapped object [0x7f1aebdf8ae0])

ERROR: Please update dosemu from git, compile it with debug
info and make a bug report with the content of ~/.dosemu/boot.log at
https://github.com/dosemu2/dosemu2/issues
Please provide any additional info you can, like the test-cases,
URLs and all the rest that fits.

Terminated with signal 11"
my english is not good and i use a translator.

**Comment by w8l13r** [comment]
ERROR: cpu exception in dosemu code outside of DPMI client!
Segmentation fault (Invalid permissions for mapped object [0x7f1aebdf8ae0])

ERROR: Please update dosemu from git, compile it with debug
info and make a bug report with the content of ~/.dosemu/boot.log at
https://github.com/dosemu2/dosemu2/issues
Please provide any additional info you can, like the test-cases,
URLs and all the rest that fits.

Terminated with signal 11

**Comment by stsp** [comment]
Your log lacks debug info.
Please update and install the
debuginfo packages for dosemu2,
or tell exactly how to reproduce the
problem.

What distribution do you have and
are you building dosemu2 from
sources or installing the packages?

**Comment by w8l13r** [comment]
hello, i am using linux mint 20 xfce 64 bit. it is installed via the update management. it is started via the terminal and then it crashes, see the above message. i can't do anything with debuginfo packages. all i can say is that it worked in february and after an update it no longer works.

**Comment by stsp** [comment]
In this case please ask someone from
mint forums to help you install debuginfo,
or invite them to this ticket directly.
Otherwise not much help can be provided.

**Comment by stsp** [comment]
You can also give me an ssh access
so that I can take a look myself.

**Comment by w8l13r** [comment]
hello, i have installed linux mint ulyana on a new disk and updated it, then i used 
sudo add-apt-repository ppa:dosemu2/ppa 
sudo apt update
installed. Unfortunately I get the same error message. Since this didn't leave me any peace, I installed linuxmint 21.3, updated everything and then installed dosemu2 as above. And here everything went through without any problems. There seems to be a problem with Linuxmint 20.4. For me the topic would be closed, as I have now switched to linuxmint 21.3. thank you anyway for your help. 
With kind regards
